mteb/abstasks/AbsTaskMultilabelClassification.py

```diff
@@ -154,14 +126,12 @@ def _evaluate_monolingual(
             )
             scores.append(scores_exp)
 
-        if self.n_experiments == 1:
-            return scores[0]
-        else:
-            avg_scores = {k: np.mean([s[k] for s in scores]) for k in scores[0].keys()}
-            std_errors = {
-                k + "_stderr": np.std([s[k] for s in scores]) for k in scores[0].keys()
-            }
-            return {**avg_scores, **std_errors}
+        avg_scores: dict[str, Any] = {
+            k: np.mean([s[k] for s in scores]) for k in scores[0].keys()
+        }
+        avg_scores["scores_per_experiment"] = scores
+
+        return avg_scores
```

Comment:
Maybe am missing sth but do we not care about the std errors anymore?

Reply:
We do, but we also care about win-rate, confidence intervals etc. so if we just keep the raw scores then people can calculate whatever metric that they want.